### ISSUE TITLE
cannot resume music by using Alexa resume voice command

### ISSUE THREAD

**Issue by benghopy** [alexa/avs-device-sdk#1799]
**IMPORTANT**: Before you create an issue, please take a look at our [Issue Reporting Guide](https://github.com/alexa/avs-device-sdk/wiki/Issue-Reporting-Guide).

## Briefly summarize your issue:

init with configJsonStreams.push_back(alexaClientSDK::afml::interruptModel::InterruptModelConfiguration::getConfig(false));
Alexa, play music
Alexa, stop
Alexa, resume but it cannot resume music when the interrupt model set to NoMixer

## What is the expected behavior?

The music can resume

## What behavior are you observing?

The music cannot resume

## Provide the steps to reproduce the issue, if applicable:

init with configJsonStreams.push_back(alexaClientSDK::afml::interruptModel::InterruptModelConfiguration::getConfig(false));
Alexa, play music
Alexa, stop
Alexa, resume but it cannot resume music when the interrupt model set to NoMixer

### What version of the AVS Device SDK are you using?
      1.20.0

### Tell us what hardware you're using:
- [ ] Desktop / Laptop
- [ ] Raspberry Pi
- Other - tell us more:

### Tell us about your OS (Type & version):
-  Linux
- [ ] MacOS 
- [ ] Raspbian Stretch
- [ ] Raspbian Jessy
- [ ] Other - tell us more:

### Please check the code about this file:
capabilities/AudioPlayer/acsdkAudioPlayer/src/AudioPlayer.cpp +1340
```C
    case PlayerActivity::FINISHED:
    case PlayerActivity::IDLE:
        // Note: can be in FINISHED or IDLE while waiting for MediaPlayer to start playing, so we fall
        // through to call @c pause() here as well.
        if (previousFocus == FocusState::NONE && !m_audioPlayQueue.empty()) {
            // If a higher priority CA (like Alarm) is in the foreground, AudioPlayer state will be
            // background We need also to start to play, the mixing behavior will decide to pause or play in
            // duck
            if (m_audioPlayQueue.front()->mixingBehavior == audio::MixingBehavior::BEHAVIOR_DUCK &&
                !m_isStartingPlayback) {
                ACSDK_DEBUG1(LX("transitionToBackgroundWhileNotPlayingAndNextItemDuckable")
                                    .d("result", "playNextItemInDuck"));
                playNextItem();
                /* !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
                 *  
                 * I think here need reutrn  
                 *
                 * !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
                 */
            }

            // if in FINISHED/IDLE/STOPPED state , NONE -> BACKGROUND transition and mixingBehavior is PAUSE
            // do not call pause on mediaplayer : this will cause PAUSE when no music is playing, messing up
            // audioplayer states we should rather wait for audioplayer to become FOREGROUND in the future
            // to play an example of this is Flash Briefing:
            // 1. Ask for Flash briefing
            // 2. AudioPlayer receives Play directive : but is in background (since SpeechSynthesizer has
            // Foreground : "here's your flash briefing" 2a. AudioPlayer should not call PAUSE through the
            // behaviorDelegate below : instead wait for FOREGROUND
            else if (m_audioPlayQueue.front()->mixingBehavior == audio::MixingBehavior::BEHAVIOR_PAUSE) {
                ACSDK_DEBUG1(LX("transitionToBackgroundWhileNotPlayingAndNextItemNotDuckable")
                                    .d("result", "delayNextItemUntilForegroundAcquired"));
                return;
            }
        } else if (m_audioPlayQueue.empty()) {
            // if in FINISHED/IDLE/STOPPED state and no audioItems in queue : no need to Duck/Pause
            ACSDK_DEBUG1(LX("transitionToBackGroundWhileNotPlaying"));
            return;
        }
```

**Comment by benghopy** [comment]
main code: This switch need return or break.
https://github.com/alexa/avs-device-sdk/blob/4bb0d23cd98f45cedb4cc0b928ee06a1267c02d2/capabilities/AudioPlayer/acsdkAudioPlayer/src/AudioPlayer.cpp#L1340

**Comment by caleighatamazon** [comment]
Hi @benghopy , thanks for reporting this. This is unfortunately a known issue in the SDK at this time. We are already tracking it internally. Thanks!

**Comment by caleighatamazon** [comment]
Closed this issue in error -- it is under investigation.

**Comment by kclchan** [comment]
This is fixed in v1.21 of the SDK.  Thanks!
